### PR TITLE
show each user type in admin page, fix catching errors in database an…

### PR DESCRIPTION
**Pull request by Carlos7979**
Nico, this branch name (feature/carlos-update-users) don't describe the actual changes beacause I have not yet worked on updating user data. The commit include the following features:
→show each user type in admin page.
→fix catching errors in database.
→actual admin user can't remove himself or herself.

### CODE REVIEW

File: database.go [hunk 1]
```diff
@@ -11,14 +11,29 @@ import (
 
 var db *sql.DB
 
+func checkErrTx(err error, tx *sql.Tx) {
```
Comment (nhatcher) [inline database.go:14]: Please remove this function

File: database.go [hunk 2]
```diff
@@ -11,14 +11,29 @@ import (
...
+	if err != nil {
+		tx.Rollback()
+		panic(err)
+	}
+}
+
+func closeTransaction(tx *sql.Tx) {
```
Comment (nhatcher) [inline database.go:21]: Please put this in a separate PR.

**Review by nhatcher — CHANGES_REQUESTED** [review]
Can you please split the PR in two?
Very nice improvement!!!!!!

File: database.go (hunk 1 above)
Comment (Carlos7979) [reply database.go:14]: Ok, if I remove this function, have I to use the newer function closeTransaction() instead of it in all those places in which are used?

File: database.go (hunk 2 above)
Comment (Carlos7979) [reply database.go:21]: This means that I have to erase this branch, I don't know how to reverse the actual change in order to separate them, can you help me with some clues?

Comment (Carlos7979) [reply database.go:21]: Sorry the above was a question, not an affirmation

Comment (Carlos7979) [reply database.go:21]: This means that have I to erase this branch?

File: database.go (hunk 1 above)
Comment (nhatcher) [reply database.go:14]: Yes, of course! But please in a different PR

File: database.go (hunk 2 above)
Comment (nhatcher) [reply database.go:21]: Let's deny this PR and then you can create two separate ones, then. That's easy.
